apps/condo/domains/banking/constants.js

```diff
@@ -2,5 +2,11 @@
  * Generated by `createschema banking.BankCategory 'name:Text;'`
  */
 
+const BANK_INTEGRATION_IDS = {
+    SBBOL: 'd94743b0-e5d5-4d06-a244-ea4b2edb8633',
```

Comment:
Why not from environment? What is that constants about?

Reply:
`BankIntegration` means available data source for transactions. Currently we've developing obtaining transactions from 2 data sources:
1. SBBOL Fintech API
2. Exported file in `1CClientBankExchange` format
For each data source a corresponding record was inserted into `BankIntegration` table, which is not supposed to be changed by any user, — they are hardcoded.